code/modules/research/machinery/_production.dm

```diff
@@ -2,19 +2,27 @@
 	name = "technology fabricator"
 	desc = "Makes researched and prototype items with materials and energy."
 	layer = BELOW_OBJ_LAYER
-	var/efficiency_coeff = 1 //Materials needed / coeff = actual.
+	/// Materials needed / coeff = actual.
+	var/efficiency_coeff = 1
 	var/list/categories = list()
 	var/datum/component/remote_materials/materials
 	var/allowed_department_flags = ALL
-	var/production_animation //What's flick()'d on print.
+	/// What's flick()'d on print.
+	var/production_animation
 	var/allowed_buildtypes = NONE
 	var/list/datum/design/cached_designs
 	var/list/datum/design/matching_designs
-	var/department_tag = "Unidentified" //used for material distribution among other things.
+	/// Used for material distribution among other things.
+	var/department_tag = "Unidentified"
 
 	var/screen = RESEARCH_FABRICATOR_SCREEN_MAIN
 	var/selected_category
 
+	/// Does this production machine have a stripe?
+	var/has_color_stripe = FALSE
```

Comment:
You don't need this, just check if `stripe_color` is null, and make the default null.

Reply:
that's fair, I'll do that

Reply:
done